keps/sig-api-machinery/20190228-priority-and-fairness.md

```diff
@@ -231,7 +231,177 @@ TBD
 
 ### Implementation Details/Notes/Constraints
 
-TBD
+##### Definitions
+
+- *BucketBinding*: a set of rules defining how we bind an inbound request w/ a bucket.
+It works similar to \[Cluster\]RoleBinding in the RBAC world.
+- *Quota*: a request must acquire one quota before its execution. Basically,
```

Comment:
It would be helpful to say here whether quota is in units of concurrency or some sort of rate.  Based on the definition of "buket", I think the answer is that quota is a unit of concurrency.